packages/integration-tests/contracts/ERC20.sol

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
What do you think about deduplicating the need to have two copies of the same contract by putting the contracts in the base of the repo as well as the waffle config?

Reply:
Good call